src/renderer/item.store.ts

```diff
@@ -1,13 +1,11 @@
-import orderBy from "lodash/orderBy";
-import { autobind, noop } from "./utils";
-import { action, computed, observable, when } from "mobx";
+import { action, computed, makeObservable, observable, toJS, when } from "mobx";
+import { noop, orderBy } from "lodash";
```

Comment:
Why are you using `lodash`'s `noop`?

Reply:
Because of `lodash/orderBy` is used.

Reply:
I think you removed all of the old line 2 instead of just the `autobind` one